### PR TITLE
Xtext 2.30

### PR DESCRIPTION
**Pull request by ashpak-shaikh**
# What Changed

Xtext 2.30

# Why

Upgrades are good

### CODE REVIEW

**Comment by ashpak-shaikh** [comment]
https://github.com/graph-quilt/graphql-xtext/issues/22

File: com.intuit.graphql.parent/com.intuit.graphql.ide/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,14 +2,14 @@ Manifest-Version: 1.0
 Automatic-Module-Name: com.intuit.graphql.ide
 Bundle-ManifestVersion: 2
 Bundle-Name: com.intuit.graphql.ide
-Bundle-Vendor: Intuit
+Bundle-Vendor: My Company
```
Comment (CNAChino) [inline com.intuit.graphql.parent/com.intuit.graphql.ide/META-INF/MANIFEST.MF:5]: should this be Intuit?

File: com.intuit.graphql.parent/com.intuit.graphql.target/com.intuit.graphql.target.target [hunk 2]
```diff
@@ -8,32 +8,37 @@
...
 			<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
-			<!-- newer Eclipse versions need Java 11 to run the Maven Tycho build -->
-			<repository location="https://download.eclipse.org/releases/2020-06"/>
+			<repository location="https://download.eclipse.org/releases/2023-03"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 			<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/modeling/emft/mwe/updates/releases/2.12.0/"/>
+			<repository location="https://download.eclipse.org/modeling/emft/mwe/updates/releases/2.14.0/"/>
```
Comment (CNAChino) [inline com.intuit.graphql.parent/com.intuit.graphql.target/com.intuit.graphql.target.target:15]: just curious if this should be 2.30.0 as well?

File: com.intuit.graphql.parent/com.intuit.graphql.tests/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -2,14 +2,15 @@ Manifest-Version: 1.0
 Automatic-Module-Name: com.intuit.graphql.tests
 Bundle-ManifestVersion: 2
 Bundle-Name: com.intuit.graphql.tests
-Bundle-Vendor: Intuit
+Bundle-Vendor: My Company
```
Comment (CNAChino) [inline com.intuit.graphql.parent/com.intuit.graphql.tests/META-INF/MANIFEST.MF:5]: Intuit?

File: com.intuit.graphql.parent/com.intuit.graphql.ui.tests/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -2,17 +2,18 @@ Manifest-Version: 1.0
 Automatic-Module-Name: com.intuit.graphql.ui.tests
 Bundle-ManifestVersion: 2
 Bundle-Name: com.intuit.graphql.ui.tests
-Bundle-Vendor: Intuit
+Bundle-Vendor: My Company
```
Comment (CNAChino) [inline com.intuit.graphql.parent/com.intuit.graphql.ui.tests/META-INF/MANIFEST.MF:5]: Intuit?

File: com.intuit.graphql.parent/com.intuit.graphql/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -2,19 +2,19 @@ Manifest-Version: 1.0
 Automatic-Module-Name: com.intuit.graphql
 Bundle-ManifestVersion: 2
 Bundle-Name: com.intuit.graphql
-Bundle-Vendor: Intuit
+Bundle-Vendor: My Company
```
Comment (CNAChino) [inline com.intuit.graphql.parent/com.intuit.graphql/META-INF/MANIFEST.MF:5]: Intuit?

File: com.intuit.graphql.parent/com.intuit.graphql.target/com.intuit.graphql.target.target (hunk 2 above)
Comment (ashpak-shaikh) [reply com.intuit.graphql.parent/com.intuit.graphql.target/com.intuit.graphql.target.target:15]: this is different than xtext

**Review by bhavinshah7 — DISMISSED** [review]

File: com.intuit.graphql.parent/com.intuit.graphql/src/com/intuit/graphql/converter/GraphQLValueConverter.java
Comment (bhavinshah7) [inline com.intuit.graphql.parent/com.intuit.graphql/src/com/intuit/graphql/converter/GraphQLValueConverter.java:1]: Is this file not needed?

Comment (ashpak-shaikh) [reply com.intuit.graphql.parent/com.intuit.graphql/src/com/intuit/graphql/converter/GraphQLValueConverter.java:1]: might be needed. good catch

**Pull request merged**